ext/mbstring/oniguruma.patch

```diff
@@ -1,61 +1,28 @@
-diff -wur oniguruma.orig/src/config.h.win32 oniguruma/src/config.h.win32
---- oniguruma.orig/src/config.h.win32	2017-07-13 17:12:45.228068900 +0200
-+++ oniguruma/src/config.h.win32	2017-07-13 16:11:19.955226200 +0200
+ ext/mbstring/oniguruma/src/config.h.win32 | 1 +
+ ext/mbstring/oniguruma/src/config.h.win64 | 1 +
+ 2 files changed, 2 insertions(+)
+
+diff --git a/ext/mbstring/oniguruma/src/config.h.win32 b/ext/mbstring/oniguruma/src/config.h.win32
+index bdbdaf25c1..9a9c43f26d 100644
+--- a/ext/mbstring/oniguruma/src/config.h.win32
++++ b/ext/mbstring/oniguruma/src/config.h.win32
 @@ -15,6 +15,7 @@
- #define SIZEOF_VOIDP 4
- #define SIZEOF_FLOAT 4
- #define SIZEOF_DOUBLE 8
+ #define SIZEOF_VOIDP 4
+ #define SIZEOF_FLOAT 4
+ #define SIZEOF_DOUBLE 8
 +#define SIZEOF_SIZE_T 4
- #define HAVE_PROTOTYPES 1
- #define TOKEN_PASTE(x,y) x##y
- #define HAVE_STDARG_PROTOTYPES 1
-diff -wur oniguruma.orig/src/config.h.win64 oniguruma/src/config.h.win64
---- oniguruma.orig/src/config.h.win64	2017-07-13 17:12:45.273605000 +0200
-+++ oniguruma/src/config.h.win64	2017-07-13 16:11:19.957231300 +0200
+ #define HAVE_PROTOTYPES 1
+ #define TOKEN_PASTE(x,y) x##y
+ #define HAVE_STDARG_PROTOTYPES 1
+diff --git a/ext/mbstring/oniguruma/src/config.h.win64 b/ext/mbstring/oniguruma/src/config.h.win64
+index 01a86c7bb9..dec7b75773 100644
+--- a/ext/mbstring/oniguruma/src/config.h.win64
++++ b/ext/mbstring/oniguruma/src/config.h.win64
 @@ -15,6 +15,7 @@
- #define SIZEOF_VOIDP 8
- #define SIZEOF_FLOAT 4
- #define SIZEOF_DOUBLE 8
+ #define SIZEOF_VOIDP 8
+ #define SIZEOF_FLOAT 4
+ #define SIZEOF_DOUBLE 8
 +#define SIZEOF_SIZE_T 8
- #define HAVE_PROTOTYPES 1
- #define TOKEN_PASTE(x,y) x##y
- #define HAVE_STDARG_PROTOTYPES 1
-diff -wur oniguruma.orig/src/regint.h oniguruma/src/regint.h
---- oniguruma.orig/src/regint.h	2017-07-13 17:12:48.686073300 +0200
-+++ oniguruma/src/regint.h	2017-07-13 17:13:28.032286100 +0200
-@@ -201,17 +201,21 @@
- } while(0)
- 
- /* sizeof(OnigCodePoint) */
-+#ifdef SIZEOF_SIZE_T
```

Comment:
This part and the others mentioned should be kept in the patch.

Reply:
This part has been fixed in upstream via kkos/oniguruma#65. Keeping it in the patch does not seem to be necessary.